### PR TITLE
Feature/ui annotation lib

### PR DESCRIPTION
**Pull request by pawelkaczmarekuk**

### CODE REVIEW

File: api/annotations/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const annotationRouter = require('./annotation');
```
Comment (alectronic0) [inline api/annotations/index.js]: you don't need this tbh
please look at
the 
`api/service/` folder we've created
https://github.com/hmcts/rpa-jui-webapp/tree/master/api/services

File: config/environments/local.config.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports = {
         s2s: 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal',
         draft_store_api: 'https://draft-store-service-aat.service.core-compute-aat.internal',
         dm_store_api: 'http://dm-store-aat.service.core-compute-aat.internal',
-        em_anno_api: 'http://em-anno-aat.service.core-compute-aat.internal',
+        em_anno_api: 'https://em-anno-aat.service.core-compute-aat.internal',
```
Comment (alectronic0) [inline config/environments/local.config.js]: please don't forget to add this to the rest of the config files

File: api/annotations/index.js (hunk 1 above)
Comment (pawelkaczmarekuk) [reply api/annotations/index.js]: are you on holiday?

Comment (alectronic0) [reply api/annotations/index.js]: maybe...

**Pull request closed without merging**